Formula/meta-package-manager.rb

```diff
@@ -0,0 +1,413 @@
+    sha256 "de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"
+  end
+
+  resource "tomli-w" do
+    url "https://files.pythonhosted.org/packages/49/05/6bf21838623186b91aedbda06248ad18f03487dc56fbc20e4db384abde6c/tomli_w-1.0.0.tar.gz"
+    sha256 "f463434305e0336248cac9c2dc8076b707d8a12d019dd349f5c1e382dd1ae1b9"
+  end
+
+  resource "typed-ast" do
+    url "https://files.pythonhosted.org/packages/07/d2/d55702e8deba2c80282fea0df53130790d8f398648be589750954c2dcce4/typed_ast-1.5.4.tar.gz"
+    sha256 "39e21ceb7388e4bb37f4c679d72707ed46c2fbf2a5609b8b8ebc4b067d977df2"
+  end
+
+  resource "types-PyYAML" do
+    url "https://files.pythonhosted.org/packages/05/6f/f19081de5ba81864f89e354560a60637822f7d6d54d9a2a907785e9b5cc4/types-PyYAML-6.0.12.9.tar.gz"
+    sha256 "c51b1bd6d99ddf0aa2884a7a328810ebf70a4262c292195d3f4f9a0005f9eeb6"
+  end
+
+  resource "types-tabulate" do
+    url "https://files.pythonhosted.org/packages/51/43/b049d9f82cc6c76f9ffa25064a75cc8cb224fa24954d83382bea42675236/types-tabulate-0.9.0.2.tar.gz"
+    sha256 "1dd4322a3a146e9073169c74278b8f14a58eb9905ca9db0d2588df408f27cac9"
+  end
+
+  resource "types-xmltodict" do
+    url "https://files.pythonhosted.org/packages/0e/3c/e478ade724ea201c6ad6203824fa5395b2302aaf0a2ed2348a247eb4ef6b/types-xmltodict-0.13.0.2.tar.gz"
+    sha256 "7a233e65aa828aac611250aa479f9e346a41a8029566ed90dac246c5888304df"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/c5/52/fe421fb7364aa738b3506a2d99e4f3a56e079c0a798e9f4fa5e14c60922f/urllib3-1.26.14.tar.gz"
+    sha256 "076907bf8fd355cde77728471316625a4d2f7e713c125f51953bb5b3eecf4f72"
+  end
+
+  resource "wcmatch" do
+    url "https://files.pythonhosted.org/packages/b7/94/5dd083fc972655f6689587c3af705aabc8b8e781bacdf22d6d2282fe6142/wcmatch-8.4.1.tar.gz"
+    sha256 "b1f042a899ea4c458b7321da1b5e3331e3e0ec781583434de1301946ceadb943"
+  end
+
+  resource "wcwidth" do
+    url "https://files.pythonhosted.org/packages/5e/5f/1e4bd82a9cc1f17b2c2361a2d876d4c38973a997003ba5eb400e8a932b6c/wcwidth-0.2.6.tar.gz"
+    sha256 "a5220780a404dbe3353789870978e472cfe477761f06ee55077256e509b156d0"
+  end
+
+  resource "xmltodict" do
+    url "https://files.pythonhosted.org/packages/39/0d/40df5be1e684bbaecdb9d1e0e40d5d482465de6b00cbb92b84ee5d243c7f/xmltodict-0.13.0.tar.gz"
+    sha256 "341595a488e3e01a85a9d8911d8912fd922ede5fecc4dce437eb4b6c8d037e56"
+  end
+
+  resource "zipp" do
+    url "https://files.pythonhosted.org/packages/ab/47/b47d02b741e0aa6f998fc80457d3dfc05cb7732ef480597c2971cbc79260/zipp-3.14.0.tar.gz"
+    sha256 "9e5421e176ef5ab4c0ad896624e87a7b2f07aca746c9b2aa305952800cb8eecb"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # Check mpm is reporting the correct version of itself.
+    assert_match(
+      /^mpm, version #{version}\n\{.*\}\n/,
+      shell_output("#{bin}/mpm --version")
+    )
```

Comment:
Maybe this:

```suggestion
    assert_match /^mpm, version #{version}\n\{.*\}\n/,
                 shell_output("#{bin}/mpm --version")
```

Reply:
Thanks for the suggestion but no, use of regex requires parenthesis:

```shell-session
$ brew test-bot --only-tap-syntax
(...)
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/meta-package-manager.rb:393:18: W: [Correctable] Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the / if it should be a division.
      assert_match /^mpm, version #{version}\n\{.*\}\n/,
```

See: https://github.com/Homebrew/homebrew-core/actions/runs/4652199961/jobs/8232262425?pr=127591#step:4:22

Reply:
Oh yea, sorry. On that note, is this regex really necessary, though? `mpm, version #{version}` actually looks fine. Same for the usage text.

As for the table outputs below, perhaps [`--output-format`](https://kdeldycke.github.io/meta-package-manager/cli-parameters.html#cmdoption-mpm-o) would help to simplify. The regexes look a bit more complicated than they should be.

Reply:
Oh good ideas! I just pushed a simplified version of the tests. I did not realized `assert_match` allowed for partial matching, so a plain string of a subset of the expected output is enough.